gto/registry.py

```diff
@@ -466,17 +484,22 @@ def _return_event(self, tag) -> TBaseEvent:
     @staticmethod
     def _echo_git_suggestion(tag):
         echo("To push the changes upstream, run:")
-        echo(f"    git push {tag}")
+        echo(f"    git push origin {tag}")
 
-    def _delete_tags(self, tags, stdout):
+    def _delete_tags(self, tags, stdout, auto_push: bool):
         tags = list(tags)
+        print()
```

Comment:
You have these prints across the code. Is this the debug output you forgot to remove?

Reply:
yes :/ I should really find out why the debugger does not work. I removed them now, thanks